packages/atomic/playwright-utils/base-fixture.ts

```diff
@@ -19,3 +20,13 @@ export const makeAxeBuilder: Parameters<
 
   await use(makeAxeBuilder);
 };
+
+base.beforeEach(async ({page}) => {
+  await page.route(/commerce\/v2\/search/, cacheHandler);
```

Comment:
We also currently have playwright tests that call non-commerce API endpoints, so we'd have to modify this accordingly.